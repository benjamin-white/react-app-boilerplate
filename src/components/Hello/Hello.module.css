.Hello {
  & h1 {
    margin-bottom: 0.25em;
    font-size: 3rem;
  }
}
