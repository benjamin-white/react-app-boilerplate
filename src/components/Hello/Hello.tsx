import { HelloProps } from './types'
import styles from './Hello.module.css'

const Hello = ({ name }: HelloProps) => (
  <div className={styles.Hello}>
    <h1>{`Hello ${name}`}</h1>
  </div>
)

export default Hello
