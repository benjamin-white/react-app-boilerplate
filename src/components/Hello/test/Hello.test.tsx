import { render, screen } from '@testing-library/react'
import Hello from '../'

describe('Hello', () => {
  it('renders a Hello component', () => {
    const name = 'Human'
    const { container } = render(<Hello name={name} />)

    expect(screen.getByText(`Hello ${name}`)).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
