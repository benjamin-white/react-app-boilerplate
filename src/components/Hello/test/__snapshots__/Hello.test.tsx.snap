// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Hello > renders a Hello component 1`] = `
<div
  class="_Hello_dcff92"
>
  <h1>
    Hello Human
  </h1>
</div>
`;
